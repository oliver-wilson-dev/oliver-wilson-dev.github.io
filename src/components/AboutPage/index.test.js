import React from 'react';
import { shallow } from 'enzyme';
import About from '.';

jest.mock('../../containers/Page', () => {
  const SubPage = ({ children }) => <div>{children}</div>;

  return SubPage;
});

const render = () => shallow(<About />);

describe('About', () => {
  it('should render correctly', () => {
    expect(render()).toMatchSnapshot();
  });
});
