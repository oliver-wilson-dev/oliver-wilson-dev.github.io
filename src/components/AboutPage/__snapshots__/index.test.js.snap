// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render correctly 1`] = `
<SubPage>
  <h1
    className="pageHeader"
  >
    About Page
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Sem viverra aliquet eget sit amet. Risus pretium quam vulputate dignissim suspendisse in est ante in. Mattis rhoncus urna neque viverra justo nec ultrices dui. Mauris cursus mattis molestie a iaculis at erat pellentesque adipiscing. A diam sollicitudin tempor id eu nisl nunc mi. Nisi lacus sed viverra tellus in hac habitasse platea dictumst. Velit sed ullamcorper morbi tincidunt ornare. Ultricies integer quis auctor elit sed. Pellentesque id nibh tortor id aliquet lectus proin nibh. Vitae tempus quam pellentesque nec nam aliquam sem. Lectus urna duis convallis convallis tellus id interdum velit laoreet. Aliquam sem fringilla ut morbi tincidunt augue. Scelerisque viverra mauris in aliquam sem. Augue neque gravida in fermentum et sollicitudin. Vestibulum morbi blandit cursus risus at ultrices. Ornare suspendisse sed nisi lacus sed viverra tellus in hac. Imperdiet nulla malesuada pellentesque elit eget gravida cum.
  </p>
  <p>
    Eros donec ac odio tempor orci dapibus ultrices. Eu nisl nunc mi ipsum faucibus vitae aliquet nec ullamcorper. Lorem mollis aliquam ut porttitor. Cum sociis natoque penatibus et magnis dis parturient montes. Fermentum dui faucibus in ornare. Tincidunt dui ut ornare lectus sit amet est placerat. Vel facilisis volutpat est velit egestas dui id ornare arcu. Arcu felis bibendum ut tristique et egestas quis. Et ligula ullamcorper malesuada proin libero nunc consequat interdum. Tempor nec feugiat nisl pretium fusce id. Viverra tellus in hac habitasse platea dictumst vestibulum. Natoque penatibus et magnis dis parturient montes nascetur ridiculus. Nulla aliquet enim tortor at. Amet nisl purus in mollis nunc sed id semper.
  </p>
  <p>
    Massa eget egestas purus viverra accumsan in nisl nisi. Dui id ornare arcu odio ut sem. Consequat semper viverra nam libero justo laoreet. Mollis aliquam ut porttitor leo a diam sollicitudin tempor. Senectus et netus et malesuada. Dignissim enim sit amet venenatis. Felis donec et odio pellentesque diam volutpat commodo sed egestas. Fermentum leo vel orci porta non pulvinar. Dictum fusce ut placerat orci nulla. Gravida rutrum quisque non tellus orci. Eu feugiat pretium nibh ipsum. Sodales ut etiam sit amet nisl purus in mollis. Odio pellentesque diam volutpat commodo sed.
  </p>
  <p>
    Etiam erat velit scelerisque in dictum non consectetur. Vel turpis nunc eget lorem dolor sed viverra. Dictumst quisque sagittis purus sit amet. Cras adipiscing enim eu turpis egestas pretium aenean pharetra. Convallis convallis tellus id interdum velit laoreet. Gravida dictum fusce ut placerat orci nulla. Arcu cursus vitae congue mauris rhoncus aenean vel elit. Adipiscing elit duis tristique sollicitudin nibh sit. Aliquet nibh praesent tristique magna. Et ultrices neque ornare aenean euismod elementum. Semper auctor neque vitae tempus quam pellentesque nec nam aliquam. Et malesuada fames ac turpis egestas integer. Convallis convallis tellus id interdum velit laoreet id. At volutpat diam ut venenatis tellus in metus vulputate eu. Purus in mollis nunc sed. At volutpat diam ut venenatis tellus in metus vulputate.
  </p>
  <p>
    Vel elit scelerisque mauris pellentesque pulvinar pellentesque. Nunc eget lorem dolor sed viverra ipsum nunc. Laoreet id donec ultrices tincidunt arcu. Cursus turpis massa tincidunt dui ut ornare lectus sit. Nulla facilisi etiam dignissim diam quis enim. Leo vel orci porta non pulvinar neque laoreet suspendisse. Habitant morbi tristique senectus et netus et malesuada fames ac. Integer enim neque volutpat ac tincidunt vitae semper. Nec feugiat nisl pretium fusce id velit ut tortor pretium. Duis tristique sollicitudin nibh sit amet commodo nulla facilisi. Morbi quis commodo odio aenean sed adipiscing diam.
  </p>
</SubPage>
`;
